eng/pipelines/runtimelab-official.yml

```diff
@@ -0,0 +1,60 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - feature/*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - '**.md'
+    - eng/Version.Details.xml
+    - .devcontainer/*
+    - .github/*
+    - docs/*
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - THIRD-PARTY-NOTICES.TXT
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+  parameters:
+    templatePath: 'templates-official'
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: TeamName
+    value: dotnet-core
+extends:
+  template: /eng/pipelines/common/templates/pipeline-with-resources.yml@self
+  parameters:
+    isOfficialBuild: true
+    stages:
+    - stage: Build
+      jobs:
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          templatePath: 'templates-official'
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          buildConfig: Release
+          platforms:
+          - windows_arm64
```

Comment:
`windows_x64` ?

Reply:
I will add that.

Reply:
Note that we don't really need `windows_arm64` here (and probably don't want it, since it doesn't get tested on PR CI).